spec/migrations/20170202195228_migrate_ansible_tower_configuration_manager_settings_to_automation_manager_spec.rb

```diff
@@ -0,0 +1,95 @@
+require_migration
+
+describe MigrateAnsibleTowerConfigurationManagerSettingsToAutomationManager do
+  let(:settings_change_stub) { migration_stub(:SettingsChange) }
+
+  migration_context :up do
+    it 'changes the key of /ems_refresh/ansible_tower_configuration% to /ems_refresh/ansible_tower_automation%' do
+      s1 = settings_change_stub.create!(
+        :key   => '/ems_refresh/ansible_tower_configuration_abc/abc',
```

Comment:
sorry, i'm confused. what is the full *real* paths? Isn't this already the full path?